controllers/admin/AdminLocalizationController.php

```diff
@@ -208,6 +209,19 @@ public function postProcess()
                 }
             }
         }
+		
+        // iPresta
+        if (Tools::isSubmit('submitLocalizationRTL')) {
+            $backOffice = Tools::getValue('generate_bo');
+            $front = Tools::getValue('generate_fo');
+            if (!$front && !$backOffice)
```

Comment:
It would be more logical to directly test `Tools::getValue(...)` (prevent the variable allocation) and reverse the `if`(first test the true condition, do the things and `else` add an error